src/assets/stylesheets/rpf_design_system/colours.scss

```diff
@@ -134,3 +134,9 @@ $rpf-grey-600: #4a4d59;
 
 $rpf-text-secondary: #4d575b;
 $rpf-text-secondary-dark: #cccccc;
+
+// Syntax highlighting
+$rpf-syntax-blue: #1498D0;
+$rpf-syntax-green: #6CE68D;
```

Comment:
Think the designs go for `#74C378`

Reply:
Will take a look in the code block ticket